<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BillingPage</class>
 <widget class="QWidget" name="BillingPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Billing Management</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item alignment="Qt::AlignmentFlag::AlignLeft">
    <widget class="QPushButton" name="backButton">
     <property name="styleSheet">
      <string notr="true">
          QPushButton {
            background-color: #f8f9fa;
            border: 1px solid #dadce0;
            color: #3c4043;
            padding: 8px 16px;
            border-radius: 4px;
          }
          QPushButton:hover {
            background-color: #e8eaed;
            border: 1px solid #d2e3fc;
          }
          QPushButton:pressed {
            background-color: #d2e3fc;
          }
        </string>
     </property>
     <property name="text">
      <string>Back to Dashboard</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Add New Bill</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Patient ID:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="patientIdInput"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Date:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDateEdit" name="dateInput">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Treatment Cost ($):</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="treatmentCostInput"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Insurance Coverage (%):</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="insuranceCoverageInput">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Description:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QTextEdit" name="descriptionInput"/>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="addBillButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Add Bill</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="billsTable"/>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Payment Summary</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Total Pending:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="totalPendingLabel">
        <property name="text">
         <string>$0.00</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Total Paid:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="totalPaidLabel">
        <property name="text">
         <string>$0.00</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="calculateButton">
       <property name="text">
        <string>Calculate Totals</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="markPaidButton">
       <property name="text">
        <string>Mark as Paid</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
